frontend/src/app/shared/components/svg-icons/goggles.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
```

Comment:
We can't use the current `svg-images-component.html` ?

Reply:
Yes

Reply:
yes absolutely. I didn't realize what that component did when I checked it earlier - I thought it was just for the bitcoin/liquid network logos.